packages/data-broker/src/services/dhis/translators/DhisTranslator.ts

```diff
@@ -207,19 +207,33 @@ export class DhisTranslator {
     return translateElementKeysInEventAnalytics(eventAnalytics, dataElementToSourceCode);
   };
 
+  // Override metadata code by data element code
   public translateInboundDataElements = (
-    dataElements: DhisMetadataObject[],
-    dataSources: DataElement[],
+    metadata: DhisMetadataObject[],
+    dataElements: DataElement[],
```

Comment:
The naming of these parameters is a little confusing, since the first argument is data elements and the second is data sources.

Reply:
Yeah it was. That's one of the reasons why I want to rename it properly. Another reason is we don't have `dataSources` anymore.